docs/source/configuration/system.md

```diff
@@ -25,23 +28,23 @@ that do not already support specifying the runtime
 
 ### Set the Python version
 
-Add the line `python-x.y` in `runtime.txt` to run the repository with Python version x.y.
+Add the line `python-x.y` in `runtime.txt` to run the repository with Python version `x.y`.
 See our [Python2 example repository](https://github.com/binder-examples/python2_runtime/blob/HEAD/runtime.txt).
 
 ### Set the R version
 
-Add the line `r-<RVERSION>-<YYYY>-<MM>-<DD>` in `runtime.txt` to run the repository with R version `RVERSION` and libraries from a `YYYY-MM-DD` snapshot of the [Posit Package Manager](https://packagemanager.posit.co/client/#/repos/2/overview).
+Add the line `r-x.y-YYYY-MM-DD` in `runtime.txt` to run the repository with R version `x.y` and libraries from a `YYYY-MM-DD` snapshot of the [Posit Package Manager](https://packagemanager.posit.co/client/#/repos/2/overview).
 
-`RVERSION` can be set to 3.4, 3.5, 3.6, or to patch releases for the 3.5 and 3.6 series.
+The version of R, i.e. `x.y`, can be set to 3.4, 3.5, 3.6, or to patch releases for the 3.5 and 3.6 series.
```

Comment:
This is out of date:
https://github.com/jupyterhub/repo2docker/blob/6cf91e45d5b03f79e365cd82eda09d1178d03327/repo2docker/buildpacks/r.py#L70-L81